--- irc.js	2016-10-29 11:27:35.773082500 +0300
+++ irc_new.js	2016-10-29 11:28:14.203170300 +0300
@@ -290,7 +290,7 @@
                             } else {
                                 channel.modeParams[mode] = [param];
                             }
-                        } else {
+                        } else if (channel.modeParams[mode]) {
                             if (arr) {
                                 channel.modeParams[mode] = channel.modeParams[mode]
                                     .filter(function(v) { return v !== param[0]; });
